modules/kms/main.tf

```diff
@@ -1,15 +1,78 @@
+locals {
+  account_id        = data.aws_caller_identity.current.account_id
+  account_principal = "arn:${data.aws_partition.current.partition}:iam::${local.account_id}:root"
```

Comment:
In the specified account, this would make every IAM role and user an administrator of the key.

Reply:
Ignored